libs/editor/src/lib/components/wizard/wizard.component.ts

```diff
@@ -37,11 +50,23 @@ export class WizardComponent implements OnInit {
 
   onStepChange(step: number) {
     this.wizardService.onWizardStepChanged(step)
+
+    this.setFocus()
     this.initializeCurrentStep()
   }
 
   private initializeCurrentStep() {
     this.currentStep = this.wizardService.getCurrentStep()
     this.configuration = this.wizardService.getStepConfiguration()
   }
+
+  private setFocus() {
+    setTimeout(() => {
+      const inputEl =
+        this.wizardFieldsEl.nativeElement.getElementsByTagName('input')[0] ||
+        this.wizardFieldsEl.nativeElement.getElementsByTagName('textarea')[0]
+
+      inputEl.focus()
+    }, 100)
```

Comment:
Doesn't look like a robust solution. I guess `afterViewInit` would be more error-proof.